README.md

```diff
@@ -1,2 +1,351 @@
+
+The format of ITN DID conform to the [W3C DID Core specification](https://www.w3.org/TR/did-core/). The W3C DID Core specification does not specify how a DID is generated, and leaves it up to the implementation provided it ensures uniqueness to a high degree of confidence.
+
+**Note:** Format section ensures the ITN DID Format complies with [W3C DID Method Syntax](https://w3c.github.io/did-core/#method-syntax)
+
+### Operations
+
+Identity Protocol provides methods to manage DIDs.
+
+#### Create
+
+Description: Based on the DID data schema context file and DID method an ITN process anchors a DID and its DID document and delivers it to the requestor.
+Constraints: Can only be invoked by the DID controller(s).
+
+Requirements:
+
+- `creator` input parameter can be a DID String or a DID document.
+- The provided DID MUST be compliant with the did:itn method.
+- The provided DID document MUST be compliant with the DID V1.0 DID document specification.
+
+`Note: Question to Umed -- as per below comments - if its just DID string - how & when DID Document is created then??`
+
+---
+
+#### Resolve
+
+Description: Based on the DID data schema context file and DID method an ITN process delivers a DID document in accordance with the W3C DID standard.
+Constraints: Any entity should be able to invoke the process.
+
+Requirements:
+
+- A DID string MUST be provided as an input parameter.
+- The provided DID MUST be compliant with the did:itn method.
+
+**example:**
+
+```typescript
+await agent.resolveDIDDoc("did:itn:NL28Bubaa8xUmhJAARNkNS")
+```
+
+Response to above code will be a DID Document as below:
+
+```json
+{
+  "didDocument": {
+    "@context": [
+      "https://www.w3.org/ns/did/v1"
+    ],
+    "assertionMethod": [
+      "did:itn:NL28Bubaa8xUmhJAARNkNS#z6MkiYmb7ncfDAAhcxtAAGsm1oySq4KYJtsi2wHaoKAnmVXC"
+    ],
+    "authentication": [
+      "did:itn:NL28Bubaa8xUmhJAARNkNS#z6MkiYmb7ncfDAAhcxtAAGsm1oySq4KYJtsi2wHaoKAnmVXC"
+    ],
+    "id": "did:itn:NL28Bubaa8xUmhJAARNkNS",
+    "keyAgreement": [
+      {
+        "id": "did:itn:NL28Bubaa8xUmhJAARNkNS#z6LSd76Tk2UJ3RVxcTjYjZ8TfvHMQgZG5HmbKPmCidPXyTGZ",
+        "type": "X25519KeyAgreementKey2020",
+        "controller": "did:itn:NL28Bubaa8xUmhJAARNkNS",
+        "publicKeyMultibase": "z6LSd76Tk2UJ3RVxcTjYjZ8TfvHMQgZG5HmbKPmCidPXyTGZ"
+      }
+    ],
+    "verificationMethod": [
+      {
+        "id": "did:itn:NL28Bubaa8xUmhJAARNkNS#z6MkiYmb7ncfDAAhcxtAAGsm1oySq4KYJtsi2wHaoKAnmVXC",
+        "type": "Ed25519VerificationKey2020",
+        "controller": "did:itn:NL28Bubaa8xUmhJAARNkNS",
+        "publicKeyMultibase": "z6MkiYmb7ncfDAAhcxtAAGsm1oySq4KYJtsi2wHaoKAnmVXC"
+      }
+    ]
+  },
+  "didDocumentMetadata": {
+    "created": "2023-11-10T15:20:59Z",
+    "updated": "2023-11-10T15:20:59Z",
+    "deactivated": false,
+    "recoveryCommitment": "99cdefca761b58664641d3f0b89b4f1e04ad833b7a24a726e3424bc6c17f39a0"
+  },
+  "didResolutionMetadata": { }
+}
+```
+
+---
```

Comment:
can be removed

Reply:
sample did doc should no be listed under a method but sepaeately

Reply:
Noted.